### PR TITLE
feat: Send Order receipt via email

### LINKED ISSUE #1176: Send Order Receipt to the User via email
### Feature Request

**Feature Description**
After an order is created there should be a feature to send the receipt of the order to the attendee.

image

**Would you like to work on the issue?**
Yes.

@Masquerade0097 Please add a dependency label as an issue for it has been created in the server repository.

### PR DESCRIPTION
**Pull request by sridharjajoo**
Fixes #1176 

Checklist:

- [x] I have checked for PMD and check-style issues 
- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream development branch.

Changes: 
1. Organizer will now be able to send receipt of the order.
~~~P.S -> there is an issue with the server because of which HTTP 403 error is thrown. Once it gets resolved the email will be sent successfully.~~~

Screenshots for the change:
![device-2018-08-01-010558](https://user-images.githubusercontent.com/20684618/43482759-3c6c697e-9527-11e8-9322-4596e94c5bc3.png)

### CODE REVIEW

File: app/src/main/res/layout/order_detail_fragment.xml [hunk 1]
```diff
@@ -164,6 +164,27 @@
...
         </FrameLayout>
 
+        <FrameLayout
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            app:layout_anchorGravity="top|end"
+            app:layout_anchor="@+id/printAction"
+            android:visibility="@{ order == null ? View.GONE : View.VISIBLE }">
+
+            <android.support.design.widget.FloatingActionButton
```
Comment (iamareebjamal) [inline app/src/main/res/layout/order_detail_fragment.xml]: Make this mini

**Comment by Masquerade0097** [comment]
Rather than creating multiple fabs we can provide these options as buttons at the bottom with image and title for print and send receipt options. The fab for printing receipt is not self-explanatory. 
@sridharjajoo Views?

**Comment by srv-twry** [comment]
> P.S -> there is an issue with the server because of which HTTP 403 error is thrown. Once it gets resolved the email will be sent successfully.

It's resolve now. You can proceed.

One thing that I noticed from the screenshot. Please do not show the option to send the receipt when an order is not `completed`. Receipts don't even exist for other cases and the server will throw an error. So only show the fab when the `order.status == "completed"`

**Comment by sridharjajoo** [comment]
@Masquerade0097 Kindly check the updated screenshot

**Comment by sridharjajoo** [comment]
@Masquerade0097 @iamareebjamal @mishuvs Please review

**Comment by sridharjajoo** [comment]
@srv-twry Please review

**Pull request merged**